packages/ui-components/src/Search/Search.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react';
+import { Button } from '../Button/index'
+
+type Props = {
+  id: string;
+  wrapperClassName?: string;
+  name?: string;
+};
```

Comment:
if you want add now onClick / onType handler for input and button and connect them.. 
but we can also do it later :)